Anytype/Sources/PresentationLayer/TextEditor/EditorPage/Views/Settings/History/VersionHistoryViewModel.swift

```diff
@@ -1,17 +1,47 @@
 import Services
 import SwiftUI
 
-struct VersionHistoryData: Identifiable {
-    let document: any BaseDocumentProtocol
-    var id: String { document.objectId }
-}
-
 @MainActor
 final class VersionHistoryViewModel: ObservableObject {
     
-    private let document: any BaseDocumentProtocol
+    @Published var versions = [VersionHistoryItem]()
+    
+    private let objectId: String
+    private var rawVersions = [VersionHistory]()
+    private var participantsDict = [String: Participant]()
+    
+    @Injected(\.historyVersionsService)
+    private var historyVersionsService: any HistoryVersionsServiceProtocol
+    @Injected(\.activeSpaceParticipantStorage)
+    private var activeSpaceParticipantStorage: any ActiveSpaceParticipantStorageProtocol
+    @Injected(\.versionHistoryDataBuilder)
+    private var versionHistoryDataBuilder: any VersionHistoryDataBuilderProtocol
+    
+    init(objectId: String) {
+        self.objectId = objectId
+    }
+    
+    func startParticipantsTask() async {
```

Comment:
Start subscriptions looks like more transparent naming to me

Reply:
Agree, fixed 🙌